.github/workflows/deploy-base.yml

```diff
@@ -18,37 +18,24 @@ on:
       SLACK_WEBHOOK:
         required: true
 
+
 jobs:
-  deploy:
-    name: Deploy
+  build:
+    name: Build/push Docker Image
     runs-on: ubuntu-latest
     permissions:
       id-token: write
       contents: read
     environment: ${{ inputs.env }}
-    env:
-      ECS_CLUSTER: skate
-      ECS_SERVICE: skate-${{ inputs.env }}
+    outputs:
+      docker-tag-suffix: ${{ steps.build-push.outputs.docker-tag-suffix }}
+      sentry-release: ${{ steps.version-ids.outputs.sentry-release }}
     steps:
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
The configure step went away, without an equivalent step needing to take its place?

Reply:
I think it was unnecessary to begin with at the start of this PR because [`build-push-ecr`](https://github.com/mbta/actions/blob/main/build-push-ecr/action.yml#L36-L41) does it for us